pkg/metrics/analysis.go

```diff
@@ -13,7 +13,7 @@ var (
 			Name:      "resource_recommendation",
 			Help:      "The containers' CPU/Memory recommended value",
 		},
-		[]string{"apiversion", "owner_kind", "namespace", "owner_name", "container", "resource", "owner_replicas"},
```

Comment:
`owner_replicas`
Why should we discard this label？

Reply:
Because I thought owner_replicas is not a value that resource recommend needs to care about, there is also no related value in recommendation CRD. 

I refactor thses code with a common method (the input parameter of the observe function was changed from RecommendationContext to Recommendation). If this field needs to be displayed, it needs to be processed separately. 

Please let me know if there is a need to keep this field or any other information I am missing.

Reply:
replicas * resource = total workload resource now
so i think it is useful here.

Reply:
OK, I will adjust it later